mapie/time_series_regression.py

```diff
@@ -32,6 +32,10 @@ class MapieTimeSeriesRegressor(MapieRegressor):
     https://arxiv.org/abs/2010.09107
     """
 
+    cv_need_agg_function = ["Subsample", "BlockBootstrap"]
+    valid_methods_ = ["enbpi"]
+    plus_like_method = ["plus", "enbpi"]
```

Comment:
Is it possible to add these attributes from the attributes of the super class? This will make valid the modifications of the MapieRegressor class to the MapieTimeSeriesRegressor class in future commits.

Here is an example:
```python
class A:
    foo = ["boo"]

class B(A):
    foo = A.foo + ["new element for B class"]

b_instance = B()
print(b_instance.foo)  # Output: ["boo", "new element for B class"]
print(B.foo)  # Output: ["boo", "new element for B class"]
```

Here is an example for an attribute of the mapie class:
```python
class MapieRegressor:
    cv_need_agg_function = ["Subsample"]

class MapieTimeSeriesRegressor(MapieRegressor):
    cv_need_agg_function = MapieRegressor.cv_need_agg_function + ["BlockBootstrap"]
```

Reply:
Thanks for the helpful comment, that makes a lot of sense. Just did this👍